RELEASE/scripts/autoscend/paths/pocket_familiars.ash

```diff
@@ -31,87 +31,16 @@ string pokefam_defaultMaximizeStatement()
 
 boolean pokefam_makeTeam()
 {
-	if(in_pokefam())
```

Comment:
Is there a reason for removing the `in_pokefam()` check?

Reply:
This is a good point.  I should and will add it back in